tdrs-backend/tdpservice/urls.py

```diff
@@ -5,18 +5,11 @@
 from django.contrib import admin
```

Comment:
We made some changes to the urls to house the app urls inside the app for easier reference.